paf-mvp-client-express/src/client-node.ts

```diff
@@ -250,7 +252,8 @@ export class ClientNode extends Node {
       res.json(response);
       next();
     } catch (e) {
-      this.logger.Error(jsonProxyEndpoints.write, e);
+      //TODO handle errors properly in a dedicated middleware
+      this.logger.Error(jsonProxyEndpoints.write, e, req.header('onekey-correlation-id'));
```

Comment:
`'onekey-correlation-id'` should be a constant shared by all nodes, to avoid typos (similar to what we do with endpoint paths for example `jsonProxyEndpoints.verify3PC`)

Reply:
Good point.